### ISSUE TITLE
Sometimes (about once a day) windows 7 HVM with seamless GUI disabled can loose control of application windows

### ISSUE THREAD

**Issue by tonsimple** [QubesOS/qubes-issues#2590]
#### Qubes OS version (e.g., `R3.2`):
3.2
#### Affected TemplateVMs (e.g., `fedora-23`, if applicable):
Windows 7 HVM with Qubes Tools installed and seamless GUI disabled

---

### Expected behavior:
Application windows are clickable and interactive

### Actual behavior:
After several hours of work, application windows (word, excel, photoshop, etc) become non-interactive

"windows interface windows" (start menu, shutdown button) are not affected.

The affected windows programs can be minimized, resized, closed by clicking on the control panel and choosing closed, but items inside window proper (word text and interface, excel cells, etc.) are off limits.

### Steps to reproduce the behavior:

Tricky.
Basically, work with word, photoshop, excel, etc. for some time, and with some probability it will happen. 

### General notes:
No workaround this time, except restart the VM :(

EDITED TO ADD:
Also happens if Seamless GUI is on sometimes (in this case, control is lost completely of all "seamlessed" windows.
Switching between seamless and non-seamless remains possible but does not in any way alleviate the issue

---

**Comment by tonsimple** [comment]
Issue also happens in seamless gui mode, happened to me right now in fact...

(switching from seamless to non-seamless and vice versa does not fix anything, issue persists, restart of windows VM is only known "cure")

**Comment by marmarek** [comment]
Just to make sure: in non-seamless mode some interface parts are working but not application windows?

**Comment by tonsimple** [comment]
Yes, exactly.
Start button is usually unaffected (which allows for a relatively graceful shutdown)
Certain elements of application window remain "workable" too (for instance, you could close an excel window using the close window button, or minimize it, but you could not select a cell in a spreadsheet, or open any of the excel's own menus)

**Comment by vercas** [comment]
When this happens, exactly one window (could even be the taskbar) remains useable with the mouse.  
Keyboard input works fine.  

Sometimes, it happens a few seconds after the VM starts. Other times, a few minutes.

**Comment by jaseg** [comment]
I do currently have a Windows 7 vm that is pretty consistently showing this issue mere minutes after boot on my system. Please tell me if there is anything I can do to debug this.

This behavior reminds me of #1599 .

**Comment by andrewdavidwong** [comment]
This issue is being closed because:

 - This issue has the "Release 3.2 updates" milestone.
 - [Qubes OS 3.2 recently reached end-of-life (EOL).](https://www.qubes-os.org/news/2019/03/28/qubes-3-2-has-reached-eol/)
 - This issue has not been updated in over a year.

If anyone believes that this issue should be reopened, please let us know in a comment here.
